src/layouts/SiteLaunchPad/SiteLaunchPad.tsx

```diff
@@ -189,14 +198,18 @@ export const SiteLaunchPad = (): JSX.Element => {
   return (
     <>
       <SiteViewHeader />
-      <VStack bg="white" w="100%" minH="100vh" spacing="2rem">
-        {title}
-        {body}
-        <RiskAcceptanceModal
-          isOpen={pageNumber === SITE_LAUNCH_PAGES.RISK_ACCEPTANCE}
-          setPageNumber={setPageNumber}
-        />
-      </VStack>
+      {shouldUseSiteLaunchFeature(siteName) ? (
+        <VStack bg="white" w="100%" minH="100vh" spacing="2rem">
+          {title}
+          {body}
+          <RiskAcceptanceModal
+            isOpen={pageNumber === SITE_LAUNCH_PAGES.RISK_ACCEPTANCE}
+            setPageNumber={setPageNumber}
+          />
+        </VStack>
+      ) : (
+        history.push(`/sites/${siteName}/dashboard`)
+      )}
```

Comment:
can we prefer using a `Redirect` here? see [here](https://v5.reactrouter.com/web/api/Redirect) for details - more declarative and intuitive rather than putting an effect in our jsx.

Reply:
good point, changed e869f3bb9c8c2432ef868054fb2c58599d686ab8!